packaging/debian/debian/copyright

```diff
@@ -0,0 +1,31 @@
+                  Open vStorage Non-Commercial License
```

Comment:
A general remark; shouldn't the healthcheck be updated to AGPL-3.0 as the other repos?